src/main/java/uk/gov/digital/ho/hocs/audit/entrypoint/CustomExportResource.java

```diff
@@ -30,9 +30,9 @@ public CustomExportResource(CustomExportService customExportService) {
                                                   @PathVariable("viewName") String viewName,
                                                   @RequestParam(name = "convertHeader", defaultValue = "false")
                                                   boolean convertHeader) {
-        response.setHeader(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=" + getFilename(viewName));
 
         try {
+            response.setHeader(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=" + getFilename(viewName));
```

Comment:
I think the try/catch needs to be inside the `getFilename` call -- putting it around the `setHeader` is too late. Even if the filename get fails, we still need to set `Content-Disposition: attachment` or the CSV will appear in-browser like a normal page, and won't trigger a download prompt.

Reply:
This try catch also wraps the customExportService.export method too so now everything is within the `try`. Previously if the export failed we would have still set the response header with the attachment content disposition which was incorrect as we wouldn't be returning any data in that scenario.